readme.md

```diff
@@ -81,188 +75,137 @@ increaseAgeBy(2)
   <pre>console.log(age)</pre>
 </details>
 
-We can make this function pure by not changing anything outside the function:
+We can make this function pure by not changing anything outside the function. Instead, we modify the parameter, which is in the scope of the function.
 
 ```js
 var age = 27
 function increaseAgeBy(age,int){
   return age += int
 }
```

Comment:
Might make it more readable to not use `age` as both a variable and argument name.